app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -8,6 +8,12 @@
     text-align: start;
   }
 
+  &.rich-govspeak {
+    strong {
+      font-weight: bold;
```

Comment:
Whitehall seems to use a different reset file without that __strong__ rule: https://github.com/alphagov/whitehall/blob/master/app/assets/stylesheets/frontend/reset/_reset.scss#L30